src/test/scala/lenthall/util/TryUtilSpec.scala

```diff
@@ -38,6 +38,28 @@ class TryUtilSpec extends FlatSpec with Matchers {
     exception.throwables.head.getMessage should be("failed")
   }
 
+  it should "sequence successful nones" in {
+    val result = sequenceOption(None, "prefix")
```

Comment:
it might be helpful for understanding to add ascriptions to these `result` variables